src/components/reporter/MitigationReviewSubmissionForm.tsx

```diff
@@ -0,0 +1,348 @@
+            Back to Contest
+          </Link>
+        </>
+      }
+      successButtonText="Submit another"
+      onSubmit={submit}
+      submitButtonText={submitButtonText}
+      className="limited-width submit-findings"
+      title={`Mitigation Review for ${sponsor}`}
+      resetForm={resetForm}
+    >
+      <fieldset className="form-field">
+        <label>Mitigation status *</label>
+        <p className="form-field__help-text">
+          <strong>Mitigation Confirmed:</strong>
+          &nbsp;the original vulnerability was successfully mitigated, and the
+          mitigation did not create any new issues
+          <br />
+          <strong>Unmitigated:</strong>
+          &nbsp;the attempted mitigation for the original issue was not fully
+          successful
+          <br />
+          <strong>New:</strong>
+          &nbsp;
+          {"a new vulnerability or bug was found - either missed in" +
+            " previous audit(s) or caused by an attempted mitigation"}
+        </p>
+        <SelectField
+          name="mitigationStatus"
+          options={mitigationStatusOptions}
+          onChange={handleChange}
+          fieldState={state.mitigationStatus}
+          required={true}
+        />
+      </fieldset>
+      <Input
+        name="mitigationOf"
+        required={
+          state.mitigationStatus === MitigationStatus.MitigationConfirmed ||
+          state.mitigationStatus === MitigationStatus.Unmitigated
+        }
+        label={
+          state.mitigationStatus === MitigationStatus.New
+            ? "Report ID of original finding (if relevant)"
+            : "Report ID of original finding"
+        }
+        placeholder="H-01"
+        value={state.mitigationOf}
+        handleChange={handleChange}
+        forceValidation={isSubmitted}
+      />
+      {state.mitigationStatus !== MitigationStatus.MitigationConfirmed && (
+        <fieldset className="form-field">
+          <label>Risk rating *</label>
+          <SelectField
+            name="risk"
+            fieldState={state.risk}
+            onChange={handleChange}
+            required={true}
+            options={[
+              {
+                label: "Low / Non-critical",
+                value: "QA (Quality Assurance)",
+              },
+              {
+                label: "Medium Risk",
+                value: "2 (Med Risk)",
+              },
+              {
+                label: "High Risk",
+                value: "3 (High Risk)",
+              },
+            ]}
+          />
+        </fieldset>
+      )}
+      {state.mitigationStatus === MitigationStatus.New && (
+        <Input
+          name="title"
+          value={state.title}
+          required={true}
+          handleChange={handleChange}
+          label="Title"
+          helpText="Summarize your findings for the bug or vulnerability. (This will be the issue title)"
+          forceValidation={isSubmitted}
```

Comment:
Is this better than using just validator on the input? I'm just curious about the reason to use forceValidation over validator?

Reply:
We actually use both, technically. The validator function, if passed in, gets called when the input loses focus. We don't need to pass in a custom validator function when the only validation check is for the value to not be empty (in that case, if we pass required, then the input automatically does a validation check when the element loses focus and shows an error if the field is empty). Where `forceValidation` comes in, is when the user tries to submit without ever clicking into the input. If that happens, we need a way to force the input component to do a validation check so that the user isn't just confused about why the form won't submit.

Reply:
Great question though! I should definitely document that somewhere, because I totally see how that would be confusing.